projects/1.20/assets/hamsters/hamsters/lang/zh_cn.json

```
@@ -0,0 +1,27 @@
+{
+  "itemGroup.hamsters.tab": "萌萌仓鼠",
+  "item.hamsters.hamsters": "仓鼠",
+
+  "entity.hamsters.hamster": "仓鼠", 
+  "item.hamsters.hamster": "仓鼠",
+
+  "tooltip.hamsters.white": "白色",
+  "tooltip.hamsters.peaches_and_cream": "白桃奶油色",
+  "tooltip.hamsters.orange": "橘色",
+  "tooltip.hamsters.grey_white": "灰白色",
+  "tooltip.hamsters.brown": "棕色",
+  "tooltip.hamsters.black_white": "黑白相间",
+  "tooltip.hamsters.black": "黑色",
+
+  "tooltip.hamsters.tamed": "已驯服",
+  "tooltip.hamsters.baby": "幼年",
+
+  "item.hamsters.hamster_spawn_egg": "仓鼠刷怪蛋",
+  "block.hamsters.tunnel": "仓鼠隧道",
+  "block.hamsters.hamster_wheel": "仓鼠跑轮",
+
+  "hamsters.subtitles.hamster.ambient": "仓鼠吱吱叫",
+  "hamsters.subtitles.hamster.hurt": "仓鼠受伤",
+  "hamsters.subtitles.hamster.death": "仓鼠死亡",
+  "hamsters.subtitles.hamster.beg": "仓鼠乞求"

```

原版中的字幕（右下角那个）是有格式的：
 - `主体：行为`，其中`主体`有时略去

如：
 - `美西螈：啾啾`
*美西螈的静息音效*
 - `怪异的噪声`
*洞穴音效*

参见：[mcw:字幕](https://zh.minecraft.wiki/w/%E5%AD%97%E5%B9%95)

```suggestion
  "hamsters.subtitles.hamster.ambient": "仓鼠：吱吱",
  "hamsters.subtitles.hamster.hurt": "仓鼠：受伤",
  "hamsters.subtitles.hamster.death": "仓鼠：死亡",
  "hamsters.subtitles.hamster.beg": "仓鼠：乞求"
```